resources/schema.edn

```diff
@@ -205,6 +207,16 @@
                               :redirectUri {:type String}}
                 :resolve     :mutation/googleLogin}
 
+  :sendLoginEmail {:type    :sendLoginEmailStatus
```

Comment:
I think it is cleaner to make this mutation only receive the `email` since the redirectUri can be generated by the server without any input from the user.

Reply:
I quite like it this way since I can point the client running on localhost to the production API

Reply:
I quite like it this setup, since I can test UI on localhost with the production server